core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1253,7 +1253,7 @@ class KafkaController(val config: KafkaConfig, zkUtils: ZkUtils, val brokerState
         // If delete topic is disabled remove entries under zookeeper path : /admin/delete_topics
         for (topic <- topicsToBeDeleted) {
           info("Removing " + getDeleteTopicPath(topic) + " since delete topic is disabled")
-          zkUtils.zkClient.delete(getDeleteTopicPath(topic))
+          zkUtils.deletePath(getDeleteTopicPath(topic))
```

Comment:
I noticed that `ZkUtils.deletePath` is a little difference than `ZkClient.delete`. It does:

```java
try {
      zkClient.delete(path)
    } catch {
      case _: ZkNoNodeException =>
        // this can happen during a connection loss event, return normally
        info(path + " deleted during connection loss; this is ok")
        false
    }
```

Not sure if this makes a difference. What do you think @junrao @onurkaraman?